### ISSUE TITLE
[BUG] Constant log errors with Seren on my Nvidia Shield Pro 2019 (possibly related to Trakt?)

### ISSUE THREAD

**Issue by Madddcow** [nixgates/plugin.video.seren#745]
**Describe the bug**
Constant errors in the log, possibly related to Seren and Trakt? This doesn't seem to happen with the identical Kodi installs on my MacBook Pro, FIreStick 4K Max and Sony Bravia TV.

**Log**
https://pastebin.ubuntu.com/p/jpgzkTKdKJ/
[kodi.log](https://github.com/nixgates/plugin.video.seren/files/8257953/kodi.log)

**Environment**
 - Nvidia Shield Pro 2019, Android 11.0.0
 - Kodi Version 19.4
 - Seren Version 2.1.9

**Comment by drinfernoo** [comment]
The errors in your log suggest your device is unable to resolve `api.trakt.tv`. Try rebooting your device.

**Comment by nutstool** [comment]
There's definitely some sort of partial issue with Seren connecting to Trakt at the moment because it's occurring for me on two separate devices. On checking the log, the error indicates too many 500 error responses from api.trakt.tv when trying to sync my collection. This started happening yesterday.

**Comment by Madddcow** [comment]
> The errors in your log suggest your device is unable to resolve `api.trakt.tv`. Try rebooting your device.

I’ve done that multiple times but as nutstool has alluded to, the number of errors is unusual enough to indicate something weird is going on.

**Comment by nutstool** [comment]
I've been looking into this further and it seems as though there may actually be a problem with my trakt account as I set up another account, added shows to the collection and this new account syncs fine. I've raised a ticket with trakt to have them look at my account because it doesn't appear to be an issue with Seren after all. I couldn't even access my trakt collection through other addons either.
